lib/helpers/rule-test-harness.js

```diff
@@ -189,10 +189,14 @@ module.exports = function generateRuleTests({
         } else if (item.verifyResults) {
           item.verifyResults(actual);
         } else {
-          let results = item.results || [item.result];
-          let expectedResults = results.map(parseResult);
+          // allow to have simple fixers tests, without result / results properties
+          let results = item.results || (item.result ? [item.result] : []);
 
-          assert.deepStrictEqual(actual, expectedResults);
+          if (results.length) {
```

Comment:
this changes necessary to have simple bad fixer asserts, and fixes issue there we don't have results and result property and we map parseResult over undefined

Reply:
I'm not sure we want this 🤔, and at the very least we should tweak it a bit to ensure you can't do:

```js
bad: [
  {
    template: 'whatever',
  }
]
```

Note no results or fixedTemplate? I definitely don't want to allow that...